ports/unix/README.md

```diff
@@ -1,17 +1,61 @@
-The Unix version
-----------------
+MicroPython Unix port
+=====================
+
+The "unix" port runs in standard Unix-like environments including Linux, BSD,
+macOS, and Windows Subsystem for Linux.
 
-The "unix" port requires a standard Unix-like environment with gcc and GNU
-make. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
-x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well as
-ARM and MIPS. Making a full-featured port to another architecture requires
+The x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well
+as ARM and MIPS. Extending the unix port to another architecture requires
 writing some assembly code for the exception handling and garbage collection.
 Alternatively, a fallback implementation based on setjmp/longjmp can be used.
 
-To build (see section below for required dependencies):
+Building
+========
```

Comment:
I feel like this should not be the same level heading as the top-level "MicroPython Unix port" heading.  Eg on stm32 the build heading is at level `------`, and then subheadings under that use `###` (level 3 heading).

Reply:
Sure, have updated.

Reply:
Thanks, looks good.